arkouda/pdarrayclass.py

```diff
@@ -26,6 +26,18 @@ def parse_single_value(msg : str) -> object:
     -------
     object numpy scalar         
     """
+    def unescape(s):
+        escaping = False
+        res = ''
+        for c in s:
+            if escaping:
+                res += c
+                escaping = False
+            elif c == '\\':
+                escaping = True
+            else:
+                res += c
+        return res
     dtname, value = msg.split(maxsplit=1)
```

Comment:
Is there a potential for an IndexError here (list only has 1 vs 2 members after split)? If so, I recommend putting this within a try...except block

Reply:
good recommendation, but since that line predates this PR, we should probably do it in a separate PR.